methods.tex

```diff
@@ -360,61 +380,38 @@ \subsection{Material Flow}
 
 \section{Scenario Specifications}
 
-The scenario specifications  are
-listed in tables \ref{tab:gen}, \ref{tab:sim_eu}, and \ref{tab:sim_france}.
+The scenario specifications  are listed in table \ref{tab:gen}.
 The reprocessing and \gls{MOX} fabrication capacity in France
 prior to 2020 is modeled after the 
-French La Hague and MELOX site \cite{schneider_spent_2008, hugelmann_melox_1999}.
+French La Hague and MELOX sites \cite{schneider_spent_2008, hugelmann_melox_1999}.
 
 
 \begin{table}[h]
     \centering
+    \caption {Simulation Specifications}
     \begin{tabularx}{\textwidth}{bb}
         \hline
         \textbf{Specification} &\textbf{ Value} \\
         \hline
-        Simulation Time & 1970-2160 \\ 
-        Reprocessed Uranium Usage &  None. Stockpile reprocessed U \\
-        Storage Residence Time & 36 months \\
-        \gls{SFR} available year & 2040 \\
+        Simulation Starts & 1970 \\
+        Simulation Ends & 2160 \\ 
         Production of \gls{ASTRID} fuel begins & 2020 \\
+        Year \glspl{SFR} become available & 2040 \\
+        Reprocessed uranium usage &  None. Stockpile reprocessed U \\
+        Used fuel cool time prior usage  & 36 months \\
+        Separation efficiency & 99.8\% \\
+        Reprocessing streams & plutonium and uranium \\
+        Reprocessing capacity before 2020 & 91.6 tonnes of \gls{UNF} per month \cite{schneider_spent_2008} \\
+        Reprocessing capacity after 2020 & $\infty$  \\
+        \gls{LWR} \gls{MOX} fab. throughput & 16.25 tonnes of \gls{MOX} per month  \cite{hugelmann_melox_1999} \\
+        \gls{ASTRID} \gls{MOX} fab. throughput & $\infty$  \\
+        \gls{LWR} \gls{MOX} fuel rep. stage  &  Used \gls{MOX} is not reprocessed. \\
+        \gls{ASTRID} \gls{MOX} fuel rep. stage & Used \gls{MOX} is reprocessed infinitely. \\  
```

Comment:
"ASTRID MOX fuel rep. stage" is confusing both because of unnecessary abbreviation and because stage is unclear. Perhaps you want to say "ASTRID MOX recycling & \infty-pass"